moderntribe/square-one
# Feature: Lefthook pre-commit merge marker

## What does this do/fix?

- Will check for merge markers and other whitespace issues before allowing you to commit your code.


## QA

- Pull this branch.
- Run `nvm use` and then `lefthook install -a`.
- Add some merge markers to a file, example below.
- Try to commit with git hooks running, it should fail and provide the errors and line numbers.

```
<<<<<<< HEAD:file.txt
Hello world
=======
Goodbye
>>>>>>> 77976da35a11db4580b80ae27e8d65caf5208086:file.txt
```

## Tests

Does this have tests?

- [x] No, this doesn't need tests because it's a git pre-commit hook.
